charts/steward/templates/podsecuritypolicy-run.yaml

```
@@ -1,4 +1,4 @@
-apiVersion: extensions/v1beta1
+apiVersion: policy/v1beta1
 kind: PodSecurityPolicy
 metadata:
   name: steward-run
```

For sorting reasons on Gardener we prefixed `00-` to ensure our PSP is used.
```suggestion
  name: 00-steward-run
```